verify.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+black diff_cover tests
+isort diff_cover tests
+python -m pytest --cov --cov-report=xml tests
+git fetch origin main:refs/remotes/origin/main
+diff-cover coverage.xml --include-untracked
+diff-quality --violations flake8 --include-untracked
+diff-quality --violations pylint --include-untracked
+doc8 README.rst
```

Comment:
I would suggest to have a newline at the end

Reply:
I'm skeptical it matters here. But guess it's nice to respect posix